### ISSUE TITLE
Error due to icon name changes in the mdi icon set

### ISSUE THREAD

**Issue by jnsebgosselin** [spyder-ide/spyder#13858]
## Problem Description

The name of 2 icons changed in the mdi icon set. This cause an error when opening spyder when using the latest version of `qtawesome` (QtAwesome 0.7.3).

![image](https://user-images.githubusercontent.com/10170372/94282050-61502c00-ff1d-11ea-9482-6cb18ddb2325.png)

### Paste Traceback/Error Below (if applicable)

```python-traceback
  File "C:\Users\User\spyder\spyder\plugins\explorer\widgets\explorer.py", line 137, in icon
    scale_factor=1.0)
  File "C:\Users\User\spyder\spyder\utils\icon_manager.py", line 455, in get_icon_by_extension_or_type
    icon_by_extension = icon(LANGUAGE_ICONS[extension], scale_factor)
  File "C:\Users\User\spyder\spyder\utils\icon_manager.py", line 420, in icon
    return qta.icon(*args, **kwargs)
  File "C:\Users\User\qtawesome\qtawesome\__init__.py", line 150, in icon
    return _instance().icon(*names, **kwargs)
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 304, in icon
    names[i]))
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 352, in _parse_options
    prefix, chars = self._get_prefix_chars(names)
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 383, in _get_prefix_chars
    raise Exception(error)
Exception: Invalid icon name "windows" in font "mdi"
Traceback (most recent call last):
  File "C:\Users\User\spyder\spyder\plugins\explorer\widgets\explorer.py", line 137, in icon
    scale_factor=1.0)
  File "C:\Users\User\spyder\spyder\utils\icon_manager.py", line 455, in get_icon_by_extension_or_type
    icon_by_extension = icon(LANGUAGE_ICONS[extension], scale_factor)
  File "C:\Users\User\spyder\spyder\utils\icon_manager.py", line 420, in icon
    return qta.icon(*args, **kwargs)
  File "C:\Users\User\qtawesome\qtawesome\__init__.py", line 150, in icon
    return _instance().icon(*names, **kwargs)
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 304, in icon
    names[i]))
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 352, in _parse_options
    prefix, chars = self._get_prefix_chars(names)
  File "C:\Users\User\qtawesome\qtawesome\iconic_font.py", line 383, in _get_prefix_chars
    raise Exception(error)
Exception: Invalid icon name "markdown" in font "mdi"
```

## Versions

* Spyder version: 4.2.0.dev0 af7d224ba
* Python version: 3.7.6 64-bit
* Qt version: 5.12.5
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.2 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.35.0;<1.0.0           :  v4.1.5+204.gaf7d224ba.dirty (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  1.0.0.dev0 (OK)
qtconsole >=4.7.7              :  4.8.0.dev0 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.10.0.dev0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.2.2 (OK)
numpy >=1.7                    :  1.19.1 (OK)
pandas >=1.1.1                 :  1.0.5 (NOK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  None (OK)

```

**Comment by ccordoba12** [comment]
> qtawesome >=0.5.7              :  1.0.0.dev0 (OK)

Thanks for the heads up @jnsebgosselin! However, this problem is not caused by 0.7.3 (which only has a very minor fix), but by the master of QtAwesome.

I think the easiest solution is to require QtAwesome >= 1.0.0 after it's released. What do you think?

**Comment by jnsebgosselin** [comment]
Oh yeah, you are right, the error occurs in `master` and not with `0.x`

> I think the easiest solution is to require QtAwesome >= 1.0.0 after it's released. What do you think?

I agree that this would be the cleanest solution.

**Comment by ccordoba12** [comment]
@jnsebgosselin, glad you agree.

@dalthviz, I think we should release 1.0 of QtAwesome right now and leave the other open PRs in that repo for later. Then you could also fix this bug. What do you think?

**Comment by dalthviz** [comment]
Sure I could do that :+1:

**Comment by ccordoba12** [comment]
This was fixed in QtAwesome 1.0.1 for both Spyder 4.1.5 and the future 4.2.0
